packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts

```diff
@@ -148,6 +149,22 @@ describe('ActorQueryOperationFilterSparqlee', () => {
       await new Promise<void>(resolve => output.bindingsStream.on('error', () => resolve()));
     });
 
+    it('Should log warning for an expressionError', async() => {
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      const logWarnSpy = jest.spyOn(<any> actor, 'logWarn');
+      // eslint-disable-next-line no-import-assign
+      Object.defineProperty(sparqlee, 'isExpressionError', { writable: true });
+      (<any> sparqlee).isExpressionError = jest.fn(() => true);
```

Comment:
Instead of overwriting this, can we create a new type of `expression` that results in an expression error?
(you may have to look into the sparqlee code to see how this can be done)

Reply:
So what you mean is just like we pass 
```JS
const erroringExpression = parse('?a + ?a');
```
right now, we should pass an expression for which we are sure runOperation would result in an ExpressionError?

Reply:
Yes, exactly!

Reply:
I'm in luck because the quoted expression is one that does this. However, the previous tactic is also used in another test. If this is what you mean I'll find a way to avoid using the tactic in the other test to.

Reply:
Well I probably won't find an expression that creates an error which is not an ExpressionError. That was stupid of me.